apps/rates-service/src/internals/rates/fetchCoin.test.ts

```diff
@@ -5,13 +5,9 @@ import { CoinUpdateInterval } from "./types";
 jest.mock('../FinnHub')
 
 it('should find a valid rate', async () => {
-
-  // We have to query NextOpenTimestamp
-  jest.setTimeout(30000);
-
   // Thu Jul 02 2020 08:35:10 GMT-0500 (Central Daylight Time)
   const lastValid = 1593696917000;
-  var now = lastValid + (24 * 60 * 60 * 1000);
```

Comment:
Had to make this edit - it turned out the next day was a holiday, so the live update (used previously) skipped right through the next day. The new version does not know that, so attempted to query data at 8:32 the next day (within the timeframe) but there is no data for that timeframe, so test failed